cypress/support/commands/common.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-undef */
+
+// Look for notification popups and disable them if present
```

Comment:
Are these for "real" notifications or for debug-type notifications?  If the latter, then this shouldn't be needed because if you run the webpack-dev-server and the rails server with `CYPRESS=true` then those debug-type notifications won't be there.